CONTRIBUTORS.txt

```diff
@@ -50,13 +54,16 @@ needs to be listed here.
 - Kyle Bashour <kylebshr@me.com>
 - Kyle Jessup <kyle@perfect.org>
 - Lev Walkin <vlm@lionet.info>
+- Liam Flynn <l.flynn@live.co.uk>
 - Ludovic Dewailly <me@ldewailly.com>
 - Markus Jais <markusjais@googlemail.com>
+- Markus Kieselmann <markus@kieselmann.eu>
 - Matt Eaton <agnosticdev@gmail.com>
 - Max Desiatov <max.desiatov@gmail.com>
 - Max Moiseev <moiseev@apple.com>
 - Moritz Lang <hi@slashmo.codes>
 - Nathan Harris <Mordil@users.noreply.github.com>
+- Nathan Harris <nathan@mordil.info>
 - Nethra Ravindran <nethra.ravindran@in.ibm.com>
 - Norio Nomura <norio.nomura@gmail.com>
 - Norman Maurer <norman_maurer@apple.com>
```

Comment:
@arukshani  if you would like me to add a proper email address here, please reply with the preferred email address.